common.gypi

```diff
@@ -264,6 +264,14 @@
 	    'cflags': [ '-m64', '-mminimal-toc' ],
 	    'ldflags': [ '-m64' ],
 	   }],
+          [ 'target_arch=="s390"', {
+            'cflags': [ '-m31' ],
+            'ldflags': [ '-m31' ],
```

Comment:
Strange but true: S/390 registers are 32 bits wide but addresses are only 31 bits wide.  `-m31` is a valid flag.